api/query/atoms_test.go

```diff
@@ -462,16 +462,51 @@ func TestStructuredQuery_triaged(t *testing.T) {
 		}}, rq)
 }
 
+func TestStructuredQuery_testlabel(t *testing.T) {
+	var rq RunQuery
+	err := json.Unmarshal([]byte(`{
+        "run_ids": [0, 1, 2],
+        "query": {
+            "exists": [{
+                "testlabel": "interop1"
```

Comment:
Can you also add a test where there are multiple labels one some tests, and a search for tests that have both labels (`&`) and another for tests that have either label (`|`)?

Reply:
I added a case for `&` abstract query. I realized that the code is yet to support multiple labels. so I will send up a follow-up PR for it